### DISCUSSION TITLE
Select Value Does Not Update if Items Change

### DISCUSSION THREAD

**Discussion by elson-currentcorp** [chakra-ui/ark#2500] [General]
https://stackblitz.com/edit/vitejs-vite-mylluo?file=src%2FApp.tsx
Click on the Change Options button. The items list updates, but the label associated with the selected value does not update.

**Comment by segunadebayo** [comment]
Good catch @elson-currentcorp.

A fix is underway [here](https://github.com/chakra-ui/zag/commit/390f2c0ae813c93ae8513c515295f8ca3be4c10b). We'll release an update shortly.

**Comment by segunadebayo** [comment]
Closing — dynamic items are supported via `createListCollection`. See the dynamic-items example in docs.
